lib/api/src/externals/table.rs

```diff
@@ -1,5 +1,7 @@
 #[cfg(feature = "js")]
 use crate::js::externals::table as table_impl;
+#[cfg(feature = "jsc")]
```

Comment:
If we introduce `jsc` as a feature, perhaps `js` needs to be renamed to `jsb` (JavascriptBrowser)

Reply:
I would prefer not to change current feature flags